test/e2e/node/pre_stop.go

```diff
@@ -92,7 +95,7 @@ func testPreStop(c clientset.Interface, ns string) {
 						PreStop: &v1.Handler{
 							Exec: &v1.ExecAction{
 								Command: []string{
-									"wget", "-O-", "--post-data=" + val, fmt.Sprintf("http://%s:8080/write", podOut.Status.PodIP),
+									"wget", "-O-", "--post-data=" + val, fmt.Sprintf("http://%s/write", podUrl),
```

Comment:
```suggestion
									"wget", "-O-", "--post-data=" + val, fmt.Sprintf("http://%s/write", podURL),
```

Reply:
I amended the commit with the changes to not mess the git history